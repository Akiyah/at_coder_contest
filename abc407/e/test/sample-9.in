10
5
84365179
63584978
310542031
12817162
73645034
258451431
185858093
336402230
757530116
14396385
5
959223565
432315360
245249028
539143978
403702602
406614700
110296991
165637124
571175581
916235701
5
853361027
349487435
523687182
747510254
447620268
120667380
710155581
221070921
146052037
78962956
5
231039975
530588909
324101694
77474051
229974231
200412556
676180871
95496353
113336826
761836157
5
430659762
235396904
976790076
610545832
328349598
165414219
995963182
564026857
248050234
727041399
5
684196092
881377747
696274241
621011060
356510849
653632852
863728508
204254549
333406059
105967492
5
637940649
393000780
730512717
707038020
118996207
300860859
799818559
693508110
193092350
399763829
5
386020741
945033984
111449334
980842504
743970548
467687012
229973560
102921216
538597184
928501347
5
351609601
863906597
704692408
707052133
621855578
280987353
36615044
376736525
433094911
323682595
5
668313974
933999501
351753196
438983441
288342313
210625502
375580678
329721156
199589112
136460346
