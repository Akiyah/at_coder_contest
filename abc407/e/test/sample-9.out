1484533791
2905561086
3184201479
1991885680
3577985709
3746587648
3568818055
3594464961
3249116317
2768630790
